libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -158,17 +175,23 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
   _onKeyboardWillShow(info?: { keyboardHeight: number }) {
     this.keyboardVisible = true;
     if (info && info.keyboardHeight) {
-      this.ionContentElement.nativeElement.style.setProperty(
-        '--keyboard-offset',
-        `${info.keyboardHeight}px`
-      );
+      this.setKeyboardOffset(`${info.keyboardHeight}px`);
     }
   }
 
   @HostListener('window:keyboardWillHide')
   _onKeyboardWillHide() {
     this.keyboardVisible = false;
-    this.ionContentElement.nativeElement.style.setProperty('--keyboard-offset', '0px');
+    this.setKeyboardOffset('0px');
+  }
+
+  private setKeyboardOffset(value: string) {
+    this.ionContentElement.nativeElement.style.setProperty('--keyboard-offset', value);
+
+    const kirbyModalFooter = this.elementRef.nativeElement.querySelector('kirby-modal-footer');
+    if (kirbyModalFooter) {
+      kirbyModalFooter.setAttribute('style', `--keyboard-offset: ${value}`);
+    }
```

Comment:
Consider changing signature to `value: number` and concatenate with `px` inside method for better encapsulationa and type safety:
```suggestion
  private setKeyboardOffset(value: number) {
    const pxValue = `${value}px`;
    this.ionContentElement.nativeElement.style.setProperty('--keyboard-offset', pxValue );

    const kirbyModalFooter = this.elementRef.nativeElement.querySelector('kirby-modal-footer');
    if (kirbyModalFooter) {
      kirbyModalFooter.setAttribute('style', `--keyboard-offset: ${pxValue}`);
    }
```